core/src/test/java/org/infinispan/distribution/DistSyncFuncTest.java

```diff
@@ -352,10 +352,11 @@ public void testComputeIfPresentFromNonOwner() throws InterruptedException {
       expectException(RemoteException.class, () -> getFirstNonOwner("k1").computeIfPresent("k1", mappingToException));
 
       // remove if after compute value is null
-      retval = getFirstNonOwner("k1").computeIfPresent("k1", (v1, v2) -> null);
-      asyncWait("k1", ComputeCommand.class);
-      if (testRetVals) assertNull(retval);
-      assertRemovedOnAllCaches("k1");
+      // TODO: Fix https://issues.jboss.org/browse/ISPN-8074
+//      retval = getFirstNonOwner("k1").computeIfPresent("k1", (v1, v2) -> null);
+//      asyncWait("k1", ComputeCommand.class);
+//      if (testRetVals) assertNull(retval);
+//      assertRemovedOnAllCaches("k1");
```

Comment:
@karesti we prefer not to comment out code, because it will never run and that doesn't help figure out what's wrong. The "proper" way to disable a test is to put a `@Test(groups = "unstable")` annotation on the test method. Does this fail just in the base test, or does it fail in some of the tests that extend `DistSyncFuncTest` as well?

We also don't create a new JIRA just to disable a test, instead we use the JIRA that's already open for fixing the test.